bootstrap/031_sqlglot.sql

```diff
@@ -9,24 +13,71 @@ as
 $$
 from sqlglot import parse_one
 import sqlglot.expressions as exp
-def test(sql, database, schema):
+def test(sql, database, schema, exclude_cte):
+    if schema is None:
+        schema = ''
+    if database is None:
+        database = ''
     try:
-        return list(f"{database if table.catalog == '' else table.catalog}.{schema if table.db == '' else table.db}.{table.name}".upper() for
+        tables = list(f"{database if table.catalog == '' else table.catalog}.{schema if table.db == '' else table.db}.{table.name}".upper() for
```

Comment:
Do you have these backwards? `database` goes to `table.db` and `schema` with `table.catalog`?

glancing at https://sqlglot.com/sqlglot/expressions.html#Table feels like you might have these backwards.